manifests/config/context.pp

```diff
@@ -0,0 +1,42 @@
+# Definition: tomcat::config::server
+#
+# Configure attributes for the Server element in $CATALINA_BASE/conf/server.xml
+#
+# Parameters
+# - $catalina_base is the base directory for the Tomcat installation.
+# - $class_name is the optional className attribute.
+# - $class_name_ensure specifies whether you are trying to set or remove the
+#   className attribute. Valid values are 'true', 'false', 'present', or
+#   'absent'. Defaults to 'present'.
+# - $address is the optional address attribute.
+# - $address_ensure specifies whether you are trying to set of remove the
+#   address attribute. Valid values are 'true', 'false', 'present', or
+#   'absent'. Defaults to 'present'.
+# - The $port to wait for shutdown commands on.
+# - The $shutdown command that must be sent to $port.
+define tomcat::config::context (
```

Comment:
See #66. I think this should probably have the option of either being in server.xml or in a separate file. The implementation in that PR has more flexibility also.